disnake/utils.py

```diff
@@ -57,6 +57,9 @@
 else:
     HAS_ORJSON = True
 
+if TYPE_CHECKING:
+    from typing_extensions import Self
```

Comment:
This is already being imported ~50 lines below, don't ask me why those imports are all the way down there, though